tests/ppx/react/expected/spreadProps.res.txt

```diff
@@ -23,3 +33,13 @@ let c1 = React.jsx(A.make, p)
 
 // reversed order
 let c2 = React.jsx(A.make, {...p, x: "x"})
+
+let c3 = ReactDOM.jsx("div", p)
```

Comment:
There's now something called `jsxNotKeyed` in rescript-react, which never seems to be called. Should it be deleted, or should it be called somewhere?

Reply:
Not sure I understand correctly.
`ReactDOM.jsx` calls both `jsxKeyed` and `jsxNotKeyed` https://github.com/rescript-lang/rescript-react/blob/9b0fe1bc548b89ad5b0ddc3acc5bfe85da1630da/src/ReactDOM.res#L1117:L1121

Reply:
Got it, thanks.